File: fibonacci/fibonacci.php
<?php

$start = microtime(true);

function fibonacci($n) {
    if ($n <= 0) {
        return 0;
    } elseif ($n == 1) {
        return 1;
    } else {
        return fibonacci($n - 1) + fibonacci($n - 2);
    }
}

$result = fibonacci(40);
echo "Fibonacci of 40: " . $result . "\n";

$elapsed = microtime(true) - $start;
echo "Time to calculate: " . $elapsed . " seconds\n";

?>

File: fibonacci/fibonacci.rs
use std::time::Instant;

fn fibonacci(n: u64) -> u64 {
    match n {
        0 => 0,
        1 => 1,
        n => fibonacci(n - 1) + fibonacci(n - 2),
    }
}

fn main() {
    let start = Instant::now();

    let result = fibonacci(40);
    println!("Fibonacci of 40: {}", result);

    let elapsed = start.elapsed();
    println!("Time to calculate: {:?}", elapsed);
}